src/tests/Microsoft.PowerFx.Core.Tests.Shared/FormatterTests.cs

```diff
@@ -269,6 +269,30 @@ public void TestPrettyPrint(string script, string expected)
             Assert.Equal(expected, result);
         }
 
+        [Theory]
+        [InlineData("x= /* test */ 1;", "x = /* test */1;")]
+        [InlineData("y =3 /* test */;", "y = 3/* test */;")]
+        [InlineData("F(ax: Number /* testttt */ , ab: Number): Number = ax* ab+y   + 2 + x;", "F(ax: Number /* testttt */ , ab: Number): Number = ax * ab + y + 2 + x;")]
+        [InlineData("X():Void = { /* test */ Notify(\"SADF\"); /* asfddsf */ Notify(\"ASDFSFD\"); /* hi */ };", "X():Void = \n{\n\t/* test */Notify(\"SADF\");\n    /* asfddsf */Notify(\"ASDFSFD\");\n    /* hi */\n};")]

```

Comment:
>{\n\t/* test */Notify [](http://example.com/codeflow?start=126&length=21)

Not sure if this is related to your changes, but shouldn't we maintain the space between `/* test */` and `Notify("SADF`?